wdl/src/bin/wdl.rs

```diff
@@ -232,6 +233,35 @@ impl AnalyzeCommand {
     }
 }
 
+/// Formats a WDL source file.
+#[derive(Args)]
+#[clap(disable_version_flag = true)]
+pub struct FormatCommand {
+    /// The path to the source WDL file.
+    #[clap(value_name = "PATH")]
+    pub path: PathBuf,
+}
+
+impl FormatCommand {
+    async fn exec(self) -> Result<()> {
+        let source = read_source(&self.path)?;
+        match format_document(&source) {
```

Comment:
Remove `format_document` and call `Document::format` (with any boilerplate
needed to get to that point in this function).

Reply:
What is the rationale here?

It seems to me that `format_document` is a useful utility function. Especially because the current `Document::format` is fallible if it's assumptions about the `Document` passing CST+AST validation are not true. I think it's good to have that validation boilerplate inside a util function rather than "hoping" whoever calls `Document::format` has performed the proper validation.